x/deployment/client/grpc_rest_test.go

```diff
@@ -0,0 +1,271 @@
+package client_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/testutil/network"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkrest "github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/ovrclk/akash/testutil"
+	"github.com/ovrclk/akash/x/deployment/client/cli"
+	"github.com/ovrclk/akash/x/deployment/types"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+
+	cfg        network.Config
+	network    *network.Network
+	deployment types.DeploymentResponse
+}
+
+func (s *IntegrationTestSuite) SetupSuite() {
+	s.T().Log("setting up integration test suite")
+
+	cfg := testutil.DefaultConfig()
+	cfg.NumValidators = 1
+
+	s.cfg = cfg
+	s.network = network.New(s.T(), cfg)
+
+	_, err := s.network.WaitForHeight(1)
+	s.Require().NoError(err)
+
+	val := s.network.Validators[0]
+
+	deploymentPath, err := filepath.Abs("./../testdata/deployment.yaml")
+	s.Require().NoError(err)
+
+	// create deployment
+	_, err = cli.TxCreateDeploymentExec(
+		val.ClientCtx,
+		val.Address,
+		deploymentPath,
+		fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+		fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+		fmt.Sprintf("--gas=%d", flags.DefaultGasLimit),
+	)
+	s.Require().NoError(err)
+
+	s.Require().NoError(s.network.WaitForNextBlock())
+
+	// get deployment
+	resp, err := cli.QueryDeploymentsExec(val.ClientCtx.WithOutputFormat("json"))
+	s.Require().NoError(err)
+
+	var out *types.QueryDeploymentsResponse = &types.QueryDeploymentsResponse{}
```

Comment:
mind using the `out := &types.QueryDeploymentsResponse{}` form for all of these from now on?